content/applications/finance/documents.rst

```diff
@@ -0,0 +1,127 @@
+:show-content:
+
+=========
+Documents
+=========
+
+**Odoo Documents** allows you to store, view and manage files within Odoo.
+
+You can upload any type of file (max 64MB per file on Odoo Online/SaaS), and organize them in
+folders known as workspaces.
+
+.. seealso::
+   - `Odoo Documents: product page <https://www.odoo.com/app/documents>`_
+
+Workflow actions
+================
+
+Workflow actions are automated actions you can create and customize at the level of each
+workspace. They appear next to a file whenever it meets the criteria you set. That way you can,
+for example, add tags to a file or move it to another workspace with a single click. Workflow
+actions help you streamline the management of your documents and your overall business operations.
+
+Create workflow actions
+-----------------------
+
+To create workflow actions, go to :menuselection:`Documents --> Configuration --> Workspaces` and
+select the workspace where the action should apply. Click on the *Actions* smart button, and then
+on *Create*.
+
+.. image:: documents/access-workflow-actions.png
+   :align: center
+   :alt: Workflow actions smart button in Odoo Documents
+
+.. note::
+   An action applies to all *Child Workspaces* under the *Parent Workspace* you selected (i.e., to
+   all subfolders under the selected folder).
```

Comment:
If "workspaces"  means "folders" I'd rather remove the "i.e." between brackets.
When I first read it, I was uncertain to which folder you were referring to, then I understood it must be proper folders, somehow different from the workspaces.

To me it isn't exactly true...
Workspaces act more like "filters" since parent workspaces also display the files in the child workspaces.

If you feel like explaining workspaces, I would do it as a separate topic before the title "Workflow actions". (maybe another PR? So we can already merge this content :) )

Reply:
Indeed, we could explain the concept of workspaces acting like filters (plus maybe a bit more info) before going into workflow actions - in another PR :)